lib/storage/providers/SQLiteStorage.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * The SQLiteStorage provider stores everything in a key/value store by
+ * converting the value to a JSON string
+ */
+import {open} from 'react-native-quick-sqlite';
+import _ from 'underscore';
+
+const DB_NAME = 'Expensify-new-db';
+const db = open({name: DB_NAME});
+
+db.execute('CREATE TABLE IF NOT EXISTS magic_map (record_key TEXT NOT NULL PRIMARY KEY , valueJSON JSON NOT NULL) WITHOUT ROWID;');
+
+// All of the 3 pragmas below were suggested by SQLite team.
+// You can find more info about them here: https://www.sqlite.org/pragma.html
+db.execute('PRAGMA CACHE_SIZE=-20000;');
+db.execute('PRAGMA synchronous=NORMAL;');
+db.execute('PRAGMA journal_mode=WAL;');
```

Comment:
Why WAL and not WAL2? I thought WAL2 was better overall, but maybe it only matters when you have multiple threads reading/writing in the DB?

Reply:
quick-sqlite doesn't support WAL2 I would need to modify that lib and will do but let's not do it in this pr.

Reply:
Thanks for the context. No need to add it for now, I assume it won't make much of a difference in this case anyway, so if it was available was worth the try, but not otherwise.